models/stg_sap.yml

```diff
@@ -0,0 +1,2158 @@
+      - name: bet39
+        description: Wage type amount for payments
+      - name: bet40
+        description: Wage type amount for payments
+      - name: bsgrd
+        description: Capacity utilization level
+      - name: cpind
+        description: Planned compensation type
+      - name: divgv
+        description: Working hours per payroll period
+      - name: ein01
+        description: Time/measurement unit
+      - name: ein02
+        description: Time/measurement unit
+      - name: ein03
+        description: Time/measurement unit
+      - name: ein04
+        description: Time/measurement unit
+      - name: ein05
+        description: Time/measurement unit
+      - name: ein06
+        description: Time/measurement unit
+      - name: ein07
+        description: Time/measurement unit
+      - name: ein08
+        description: Time/measurement unit
+      - name: ein09
+        description: Time/measurement unit
+      - name: ein10
+        description: Time/measurement unit
+      - name: ein11
+        description: Time/measurement unit
+      - name: ein12
+        description: Time/measurement unit 
+      - name: ein13
+        description: Time/measurement unit
+      - name: ein14
+        description: Time/measurement unit
+      - name: ein15
+        description: Time/measurement unit
+      - name: ein16
+        description: Time/measurement unit
+      - name: ein17
+        description: Time/measurement unit
+      - name: ein18
+        description: Time/measurement unit
+      - name: ein19
+        description: Time/measurement unit
+      - name: ein20
+        description: Time/measurement unit
+      - name: ein21
+        description: Time/measurement unit
+      - name: ein22
+        description: Time/measurement unit
+      - name: ein23
+        description: Time/measurement unit
+      - name: ein24
+        description: Time/measurement unit
+      - name: ein25
+        description: Time/measurement unit
+      - name: ein26
+        description: Time/measurement unit
+      - name: ein27
+        description: Time/measurement unit
+      - name: ein28
+        description: Time/measurement unit
+      - name: ein29
+        description: Time/measurement unit
+      - name: ein30
+        description: Time/measurement unit
+      - name: ein31
+        description: Time/measurement unit
+      - name: ein32
+        description: Time/measurement unit
+      - name: ein33
+        description: Time/measurement unit
+      - name: ein34
+        description: Time/measurement unit
+      - name: ein35
+        description: Time/measurement unit
+      - name: ein36
+        description: Time/measurement unit 
+      - name: ein37
+        description: Time/measurement unit
+      - name: ein38
+        description: Time/measurement unit
+      - name: ein39
+        description: Time/measurement unit
+      - name: ein40
+        description: Time/measurement unit
```

Comment:
Are these all intended to be the same description?

Reply:
Yes, see other screenshots with similar comments.